nav2_bringup/package.xml

```diff
@@ -9,28 +9,10 @@
   <license>Apache License 2.0</license>
 
   <buildtool_depend>ament_cmake</buildtool_depend>
-
-  <build_depend>rclcpp</build_depend>
-  <build_depend>navigation2</build_depend>
-  <build_depend>builtin_interfaces</build_depend>
-  <build_depend>rosidl_default_generators</build_depend>
-  <build_depend>launch_ros</build_depend>
-
-  <exec_depend>launch_ros</exec_depend>
-  <exec_depend>rclcpp</exec_depend>
-  <exec_depend>builtin_interfaces</exec_depend>
-  <exec_depend>rosidl_default_runtime</exec_depend>
+  
+<exec_depend>launch_ros</exec_depend>
   <exec_depend>navigation2</exec_depend>
 
-  <test_depend>ament_lint_common</test_depend>
-  <test_depend>ament_lint_auto</test_depend>
```

Comment:
Might also want to keep the linters. (`ament_lint_common` and `ament_lint_auto`)There's not a lot to lint in this package but it should at least check the package.xml and Cmakelists files.